src/front/wgsl/mod.rs

```diff
@@ -4269,11 +4200,6 @@ impl Parser {
                             }
                         }
                     }
-                    for size in workgroup_size.iter_mut() {
-                        if *size == 0 {
-                            *size = 1;
-                        }
-                    }
```

Comment:
So the difference in behavior here is that if `parse_generic_non_negative_int_literal` returns `0`, the old code will correct it to a `1`, whereas the new code leaves it zero. Is that change intentional?

Reply:
Okay, I see - nice catch.